qiskit/visualization/circuit_visualization.py

```diff
@@ -55,7 +55,8 @@ def circuit_drawer(circuit,
                    reverse_bits=False,
                    justify=None,
                    vertical_compression='medium',
-                   idle_wires=True):
+                   idle_wires=True,
+                   with_layout=False):
```

Comment:
Is there a reason to not make this the default? Presumably, if we've transpiled a circuit and have a layout we'd want to see that mapping by default in the visualization. It feels more like an opt-out kind of thing than an opt-in.

Reply:
It's to avoid changing previous behavior. I think the default can be changed in the future.

Reply:
I actually don't think printing the layout by default when we add this would be as disruptive as changing the default value after we add it. Layout labels weren't an option before and I don't think it'll will be disruptive for existing users (as long as we clearly document it) if we print them by default now that it's available. But, if we choose a direction and then decide to change the default value of the kwarg after we define the behavior that would be quite disruptive because people will start relying on that documented behavior.

Reply:
okey.. flipping that.